pkg/apiserver/installer/conversion.go

```diff
@@ -63,15 +63,7 @@ func ConvertURLValuesToV1beta2MetricListOptions(in *url.Values, out *cmv1beta2.M
 
 // RegisterConversions adds conversion functions to the given scheme.
 func RegisterConversions(s *runtime.Scheme) error {
-	if err := s.AddConversionFunc((*url.Values)(nil), (*cmv1beta1.MetricListOptions)(nil), func(a, b interface{}, scope conversion.Scope) error {
+	return s.AddConversionFunc((*url.Values)(nil), (*cmv1beta1.MetricListOptions)(nil), func(a, b interface{}, scope conversion.Scope) error {
 		return ConvertURLValuesToV1beta1MetricListOptions(a.(*url.Values), b.(*cmv1beta1.MetricListOptions), scope)
-	}); err != nil {
-		return err
-	}
-	if err := s.AddConversionFunc((*url.Values)(nil), (*cmv1beta2.MetricListOptions)(nil), func(a, b interface{}, scope conversion.Scope) error {
-		return ConvertURLValuesToV1beta2MetricListOptions(a.(*url.Values), b.(*cmv1beta2.MetricListOptions), scope)
```

Comment:
I don't think we want to remove that, do we?

Reply:
you are right, we don't want, I thought that it was repited but they are different, nice catch!